tex/gloss-korean.ldf

```diff
@@ -0,0 +1,192 @@
+\ProvidesFile{gloss-korean.ldf}[polyglossia: module for Korean]
+
+\PolyglossiaSetup{korean}{
+    script=Hangul,
+    scripttag=hang,
+    language=Korean,
+    langtag=KOR,
+    hyphennames={english,american,usenglish,USenglish},
```

Comment:
The pull request has been merged, but wouldn’t it be better to use `nohyphenation` here?  Using English hyphenation doesn’t seem right, and won’t have any effect on Hangul anyway.